<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Owner</class>
 <widget class="QWidget" name="Owner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="ownerWidget" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>801</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
    /*background-color: #2E3648;*/
	background-color : transparent;
	font-weight: bold;
	/*color: #BDC8E2;*/
	color: #2c3e50;

	padding-left: 10px;
	padding-top: 8px;
	padding-right: 7px;
	padding-bottom: 9px;
	
	border-style: solid;
    border-width: 2px;
    border-color: aqua;
    border-radius: 20px;	

}

QLabel:hover{
	color: #2c3e50;
	border-color: #95a5a6;
    background-color: #ecf0f1;
}

</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_class">
       <property name="text">
        <string>班级：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_phoneNumber">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_sex">
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_specialized">
       <property name="text">
        <string>专业：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
